python/perf-kernels/layernorm.py

```diff
@@ -177,12 +187,17 @@ def run_benchmark(args):
         plot_name = str("layernorm-performance_" + args.dtype + "_N" + str(args.N_start) + "_M" + str(args.M_start) +
                         "-" + str(args.M_end) + "-" + str(args.M_step))
         x_names = ['M']
-    else:
-        x_vals_list = [i for i in range(args.N_start, args.N_end, args.N_step)]
+    elif (sweep_n):
+        x_vals_list = [i for i in range(args.N_start, args.N_end + 1, args.N_step)]
         mn_args = {'M': args.M_start}
         plot_name = str("layernorm-performance_" + args.dtype + "_M" + str(args.M_start) + "_N" + str(args.N_start) +
                         "-" + str(args.N_end) + "-" + str(args.N_step))
         x_names = ['N']
+    else:
+        x_vals_list.append(args.N_start)
+        x_names = ['N']
+        mn_args = {'M': args.M_start}
+        plot_name = str("layernorm-performance" + "_M" + str(args.M_start) + "_N" + str(args.N_start))
```

Comment:
[Non-blocking] I think f-string interpolation makes this piece of code more readable:
```python
plot_name = f"layernorm-performance_M{args.M_start}_N{args.N_start}"
```

Probably the same suggestion can be applied in other assignments to `plot_name` as well (lines 193, 187).